manifest/openshift4/coreos/ncp-openshift4.yaml

```diff
@@ -0,0 +1,1231 @@
+
```

Comment:
If we only support Openshift on CoreOS, no need to add sub-directory under manifest/openshift4?

Reply:
Just thought maybe we will support more OS for openshift in the future...